test/resources/prometheus.py

```diff
@@ -0,0 +1,22 @@
+from robot.libraries.BuiltIn import BuiltIn
+
+import requests
```

Comment:
Have you seen https://docs.robotframework.org/docs/different_libraries/requests ?
We even have it in the `test/requirements.txt`